shared/src/native-src/pal.h

```diff
@@ -46,15 +46,14 @@ inline fs::path GetDefaultLogDir()
 
     if (isAas)
     {
-#ifdef _WINDOWS
-        return WStr("C:\\home\\LogFiles\\datadog\\");
+#ifdef _WIN32
+        return WStr(R"(C:\home\LogFiles\datadog\)");
```

Comment:
Probably a silly question, but are the brackets _inside_ the `"` intentional?

Reply:
Yep, it's intentional: its C++ way for raw-string literal (same as @"" in C#)